lib/datasource/crate/index.ts

```diff
@@ -1,88 +1,254 @@
+import * as fs from 'fs-extra';
```

Comment:
Sorry, but use `util/fs`. I Know those are only proxies but for consistency
```suggestion
```

Reply:
Pushed a fix!